### PR TITLE
Updated round tips, added button to recieve random tip in character settings

### PR DESCRIPTION
**Pull request by carlarctg**
# About the pull request

Added more tips.

Removed redundant or outdated tips.

Removed the 'chain of command' and 'shiny medal' tips.

Added a button to get a random round tip in character settings.

# Explain why it's good for the game

> Removed the 'chain of command' and 'shiny medal' tips.

These tips don't really serve any purpose other than maybe riling up whoever read them, they're kind of just a middle finger. Especially since they appear so commonly in the 1/3rd chance meta tips section.

> Added a button to get a random round tip in character settings.

I remember watching a video on how unhelpful TF2 is to new players and how it said that the only actual mechanism new players have for learning any information in the game is the 'view random tip' button in class loadouts. 
It's not a great fact that we didn't even have that, and we don't have any other good ways to learn the game, but hey, still good.

# Testing Photographs and Procedure

<details>
<summary>Screenshots & Videos</summary>

Put screenshots and videos here with an empty line between the screenshots and the `<details>` tags.

</details>

# Changelog

:cl:
add: Added more tips of the round, and removed outdated, useless, or redundant ones.
add: Added a button to get a random round tip in character gameplay settings.
/:cl:

### CODE REVIEW

File: code/modules/client/preferences_toggles.dm [hunk 1]
```diff
@@ -409,6 +409,26 @@
...
+		"Meta" = "strings/metatips.txt"
+	)
+	var/list/tip_list = file2list(types_to_pick[picked_type])
+	if(length(types_to_pick[picked_type]))
+		message = pick(tip_list)
+	else
+		CRASH("send_tip_of_the_round() failed somewhere")
+
+	if(message)
+		to_chat(src, "<span class='purple'><b>Random Tip: </b>[html_encode(message)]</span>")
```
Comment (harryob) [inline code/modules/client/preferences_toggles.dm:427]: SPAN_PURPLE

**Review by harryob — CHANGES_REQUESTED** [review]

File: code/modules/client/client_procs.dm [hunk 2]
```diff
@@ -49,9 +49,11 @@ GLOBAL_LIST_INIT(whitelisted_client_procs, list(
 	/client/proc/toggle_clickdrag_override,
 	/client/proc/toggle_dualwield,
 	/client/proc/toggle_middle_mouse_swap_hands,
+	/client/proc/receive_random_tip,
 	/client/proc/toggle_vend_item_to_hand,
 	/client/proc/switch_item_animations,
-	/client/proc/toggle_admin_sound_types
+	/client/proc/toggle_admin_sound_types,
+	/client/proc/receive_random_tip
```
Comment (fira) [inline code/modules/client/client_procs.dm:56]: you doubled it up

File: code/modules/client/preferences_toggles.dm [hunk 3]
```diff
@@ -409,6 +409,26 @@
...
+	var/picked_type = tgui_alert(src, "What kind of tip?", "Tip Type", list("Marine", "Xenomorph", "Meta")) //no memetips for them joker imp
+	var/message
+	var/list/types_to_pick = list(
+		"Marine" = "strings/marinetips.txt",
+		"Xenomorph" = "strings/xenotips.txt",
+		"Meta" = "strings/metatips.txt"
+	)
+	var/list/tip_list = file2list(types_to_pick[picked_type])
+	if(length(types_to_pick[picked_type]))
+		message = pick(tip_list)
```
Comment (fira) [inline code/modules/client/preferences_toggles.dm:422]: you can use SAFEPICK() for this (but erroring with a CRASH on failure is still good)

File: code/modules/client/preferences_toggles.dm [hunk 4]
```diff
@@ -409,6 +409,26 @@
 		prefs.toggles_sound ^= SOUND_ADMIN_ATMOSPHERIC
 		to_chat(src, SPAN_NOTICE("You will [atmospheric_toggle ? "now" : "no longer"] hear atmospheric admin sounds."))
 
+/client/proc/receive_random_tip()
+	var/picked_type = tgui_alert(src, "What kind of tip?", "Tip Type", list("Marine", "Xenomorph", "Meta")) //no memetips for them joker imp
+	var/message
+	var/list/types_to_pick = list(
```
Comment (fira) [inline code/modules/client/preferences_toggles.dm:415]: /static/

File: strings/marinetips.txt [hunk 5]
```diff
@@ -64,3 +63,25 @@ Put .w or :w before your message to whisper. Another way to whisper is to use th
...
+Be careful! Barricades block grenades, and indeed all thrown items, from the front if they're barbed up.
+Painkillers do not stack. Only the strongest has any effect.
+Intel Officers can be put in a squad by going to CIC and requesting it.
+Any marine can perform CPR. On dead marines, this will increase the time they have until they become unrevivable.
+If you've been pounced on and your squad is unloading into the target, you can hit the 'rest' button to stay down so you don't get filled with lead after getting up.
+You can check the landing zone as a marine in the status panel.
+Functioning night vision goggles can be recharged with batteries. Broken night vision goggles can be repaired by an Engineer with a screwdriver. Not the loadout ones though, those are unfixable and unchargeable.
+You can put a pistol belt on your suit slot. (Just grab a rifle instead..)
+Alt-clicking the Squad Leader tracker lets you track your fireteam leader instead.
+Armor has a randomized reduction in effectiveness, and does not protect the digits. Take the wiki damage values as a best case scenario.
```
Comment (fira) [inline strings/marinetips.txt:81]: Could add one about crit hits too :trol:

File: strings/metatips.txt [hunk 6]
```diff
@@ -5,7 +5,9 @@ After dying, ask yourself what you did wrong and make a mental note to not make
...
 Half of getting good is knowing to be aggressive. The other half is knowing when not to be aggressive.
-Past lives have absolutely no effect on the chain of command.
 Alt-click a storage item to draw the last item in it (last non-weapon if it's a weapon belt). Middle-click a storage item to inmediately open it, and middle-click structures to attempt to vault them.
-A more shiny medal or higher xeno rank does not make you better at the game.
 Use "North, South, East, West" when referring to locations in-game rather than "up, down, left, right".
+You shouldn't ignore what your allies are up to. Sometimes they can be organizing a flank in hivemind/radio, sometimes they can be walking up behind you with a slug-loaded shotgun. Either way, it pays to be alert to what they're doing, as much to as what the enemies are.
+The Wiki (https://cm-ss13.com/wiki) is a very useful repository of information about the game, such as weapons, equipment, xenomorph castes and their strains. It may not be fully up to date the majority of the time, but the basics are usually accurate.
+As an observer, you may see how much remaining hijack time is left in the status panel.
+Embrace the suck.
```
Comment (fira) [inline strings/metatips.txt:13]: huh

**Review by fira — CHANGES_REQUESTED** [review]

**Comment by carlarctg** [comment]
erm the approvy?

**Review by stanalbatross — APPROVED** [review]
the approvy

**Pull request merged**